packages/nimble-components/src/utilities/README.md

```diff
@@ -0,0 +1,5 @@
+# Utilities
+
+Utilities APIs are intended for Nimble use only and should be considered internal.
+
+Any existing usages by clients should be considered tech debt which should be fixed by moving the utility to a nimble-components directory that is considered public and re-exporting it from a framework package.
```

Comment:
```suggestion
Utilities APIs are intended for Nimble use only and should be considered internal.
```

If we want to track the tech debt I think it should be a tech debt issue instead of in docs (frankly think someone adding to that tech debt has an equal if not lower chance of discovering it via markdown docs vs a tech debt issue).